docs/index.html

```diff
@@ -1,30 +1,32 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8">
+		<head>
+				<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>Night - Home</title>
+				<title>Night - Home</title>
 		<link rel="icon" type="image/png" href="media/night-logo-black.png"/>
 		<link rel="stylesheet" type="text/css" href="styles/main.css"/>
 		<link rel="stylesheet" type="text/css" href="styles/title.css"/>
 		<link rel="stylesheet" type="text/css" href="styles/code.css"/>
 		<link rel="stylesheet" type="text/css" href="styles/index.css"/>
-    </head>
-    <body>
+		</head>
+		<body>
 		<div id="title">
 			<script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.3/p5.min.js"></script>
 			<script src="scripts/index.js"></script>
-			
+
 			<img src="media/night-logo-black.png" height="180">
+
 			<h1>Night</h1>
 
 			<hr>
 			<p id="tag"><b>Night</b> lets you see the <b>Stars</b></p>
 
 			<div id="actions">
-				<div class="action"><a href="install.html"><h2>Install</h2></a></div>
-				<div class="action"><a href="learn.html"><h2>Learn</h2></a></div>
-				<div class="action"><h2>Extras</h2></div>
+				<div class="action"><a href="index.html"><p class="desc current-page"><b>Home</p></b></a></div>
+				<div class="action"><a href="install.html"><p class="desc"><b>Install</p></b></a></div>
+				<div class="action"><a href="learn.html"><p class="desc"><b>Learn</p></b></a></div>
+				<div class="action"><a href="extras.html"><p class="desc"><b>Extras</p></b></a></div>
```

Comment:
what is the `desc` tag for? I think the css for the `action` class has everything already